packages/quantic/force-app/main/translations/fr.translation-meta.xml

```diff
@@ -484,4 +484,44 @@
       <label>Voir moins</label>
       <name>quantic_SmartSnippetShowLess</name>
     </customLabels>
+    <customLabels>
+      <label>Expliquer pourquoi</label>
+      <name>quantic_ExplainWhy</name>
+    </customLabels>
+    <customLabels>
+      <label>Sélectionnez la raison</label>
+      <name>quantic_SelectTheReason</name>
+    </customLabels>
+    <customLabels>
+      <label>Quelle était la raison?</label>
```

Comment:
If you applied my earlier comment, then consider updating this string to "Veuillez fournir des détails" or "Veuillez fournir la raison"